Core JS/index.js

```diff
@@ -0,0 +1,12 @@
+var totalFormSubmitted = 0;
+var submittedData = [];
+
+function formData(event) {
+  event.preventDefault();
+  totalFormSubmitted++;
+  submittedData.push(document.getElementById("myForm").elements[0].value);
```

Comment:
We can perform basic validation of user input, the length of the first name should be greater than 0.
Regex validation for the name.